tests/test_cli_v1.py

```diff
@@ -41,3 +46,43 @@ def test_v1_verify_exits_with_zero():
     runner = CliRunner()
     result = runner.invoke(cli, ['verify'])
     assert result.exit_code == 0
+
+
+def _load_tree(root, replace_name=None):
+    if not replace_name:
+        replace_name = str(root)
+    return {
+        x.relative_to(root).name: x.read_text().replace(replace_name, 'ROOT').replace('\\', '/')
```

Comment:
I added path separator normalization ☝🏻

Reply:
Oh, right, I missed the separators outside the root :facepalm:, :+1: